mitosheet/mitosheet/utils.py

```diff
@@ -237,6 +212,49 @@ def df_to_json_dumpsable(
     }
 
 
+def get_row_data_array(df: pd.DataFrame) -> List[Any]:
+    """
+    Returns just the data of a dataframe in the 2d array format of [row idx][col idx]
+    """
+    json_obj = convert_df_to_json(df)
+    return json_obj['data']
+
+def convert_df_to_json(df: pd.DataFrame) -> Dict[str, Any]:
```

Comment:
This is... a fucking great function. What took us so long? Nice work pulling this out.

Reply:
Maybe name it `convert_df_to_parsed_json` though?